core/common/src/main/java/alluxio/wire/MasterAddress.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import com.google.common.base.Objects;
+import com.google.common.base.Preconditions;
+
+import java.io.Serializable;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * The address of a master.
+ */
+@NotThreadSafe
+public final class MasterAddress implements Serializable {
```

Comment:
this type looks very similar to class `Address`, can we extend `Address` to simplify the code?

Reply:
Combined MasterAddress and Address. When we need to add more fields to MasterAddress, we can create MasterAddress at that time. Thanks